<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Translate</class>
 <widget class="QMainWindow" name="Translate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Translate</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>650</width>
      <height>600</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{

	background-color:#14213d;
	border:6px solid #FFDB70;

border-radius:15px;

}
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="DestFrame">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>360</y>
       <width>581</width>
       <height>221</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame {
	border:4px solid #a7a7a7;
	border-radius: 20px;
}
QFrame:hover {
	border: 4px solid #ffdb70;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPlainTextEdit" name="txtDest">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>541</width>
        <height>171</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Gandom</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPlainTextEdit{
color: rgb(255, 255, 255);
border:none;
padding:15px 1px 2px 5px;


}</string>
      </property>
      <property name="placeholderText">
       <string>ترجمه</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="SrcFrame">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>581</width>
       <height>221</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame {
	border:4px solid #a7a7a7;
	border-radius: 20px;
}
QFrame:hover {
	border: 4px solid #ffdb70;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPlainTextEdit" name="txtSrc">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>541</width>
        <height>171</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Gandom</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPlainTextEdit{

color: rgb(255, 255, 255);
padding:15px 1px 2px 5px;
border:none;
	
}</string>
      </property>
     </widget>
     <widget class="QPushButton" name="MicButton">
      <property name="geometry">
       <rect>
        <x>545</x>
        <y>12</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: none;
border-radius: 20px;
  min-height: 7px;
  min-width: 7px;

}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../Images/mic.png</normaloff>../Images/mic.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>17</width>
        <height>17</height>
       </size>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QComboBox" name="comboDest">
     <property name="geometry">
      <rect>
       <x>53</x>
       <y>350</y>
       <width>200</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gandom</family>
       <pointsize>10</pointsize>
       <stylestrategy>PreferDefault</stylestrategy>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox{
border:2px solid #ffdb70;
	background-color: rgb(255, 255, 255);
padding-left:5px;
}
QFrame{
border:none;
padding-left:5px;
border-radius:3px;
background-color: rgb(255, 255, 255);
}</string>
     </property>
    </widget>
    <widget class="QLabel" name="Title">
     <property name="geometry">
      <rect>
       <x>32</x>
       <y>18</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>A Salamat</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
color: #a7a7a7;
border:none;</string>
     </property>
     <property name="text">
      <string>Toolbox &lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;-&lt;/span&gt; Translate</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="ExitButton">
     <property name="geometry">
      <rect>
       <x>605</x>
       <y>20</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>C:/Users/Fateme/Downloads/Icon awesome-power-off.png</normaloff>C:/Users/Fateme/Downloads/Icon awesome-power-off.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="comboSrc">
     <property name="geometry">
      <rect>
       <x>53</x>
       <y>90</y>
       <width>200</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gandom</family>
       <pointsize>10</pointsize>
       <stylestrategy>PreferDefault</stylestrategy>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox{
border:2px solid #ffdb70;
	background-color: rgb(255, 255, 255);
padding-left:5px;
}
QFrame{
border:none;
padding-left:5px;
border-radius:3px;
background-color: rgb(255, 255, 255);
}</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txtSrc</tabstop>
  <tabstop>comboSrc</tabstop>
  <tabstop>comboDest</tabstop>
  <tabstop>MicButton</tabstop>
  <tabstop>txtDest</tabstop>
  <tabstop>ExitButton</tabstop>
 </tabstops>
 <resources>
  <include location="QRC/images.qrc"/>
 </resources>
 <connections/>
</ui>
